bin/varnishd/mgt/mgt_child.c

```diff
@@ -588,12 +599,26 @@ MGT_Child_Cli_Fail(void)
 		return;
 	if (child_pid < 0)
 		return;
-	MGT_complain(C_ERR, "Child (%jd) not responding to CLI, killing it.",
-	    (intmax_t)child_pid);
-	if (MGT_FEATURE(FEATURE_NO_COREDUMP))
-		(void)kill(child_pid, SIGKILL);
-	else
-		(void)kill(child_pid, SIGQUIT);
+	if (kill_child() == 0)
+		MGT_complain(C_ERR, "Child (%jd) not responding to CLI,"
+		    " killing it.", (intmax_t)child_pid);
```

Comment:
I moved the complain under the kill so that we would know if we ran into an error, and the be able to write something slightly different on error. The wording was kept, but I guess it would be better to say "killed".